### PR TITLE
Validation tests should use source filename from coverage results

### PR DESCRIPTION
**Pull request by Godin**
As discussed in https://github.com/jacoco/jacoco/pull/709#discussion_r202812090

### CODE REVIEW

**Comment by Godin** [comment]
@marchof while we haven't yet added/merged tests for Kotlin, I did a test and this will work for them. Could you please review?

**Comment by Godin** [comment]
@marchof fixed

File: org.jacoco.core.test/src/org/jacoco/core/test/validation/SourceTest.java [hunk 1]
```diff
@@ -76,12 +76,4 @@ public void testDuplicateTag() throws IOException {
 		new Source(new StringReader(src));
 	}
 
-	@Test
-	public void testGetSourceFor() throws IOException {
-		final Source s = Source.getSourceFor("src", SourceTest.class);
-		// Here we are. $line-testGetSourceFor$
-		final String l = s.getLine(s.getLineNumber("testGetSourceFor"));
-		assertTrue(l, l.contains("Here we are."));
```
Comment (marchof) [inline org.jacoco.core.test/src/org/jacoco/core/test/validation/SourceTest.java]: Damn, I've overlooked this: There is another unused import for `assertTrue` now.

Comment (Godin) [reply org.jacoco.core.test/src/org/jacoco/core/test/validation/SourceTest.java]: @marchof fixed, also checked that there is no other unused imports in modified files :wink:

Comment (marchof) [reply org.jacoco.core.test/src/org/jacoco/core/test/validation/SourceTest.java]: 👍

**Pull request merged**